src/GitHub.Api/Metrics/UsageTracker.cs

```diff
@@ -154,10 +156,7 @@ private void RunTimer(int seconds)
 
         private Usage GetCurrentUsage(UsageStore usageStore)
         {
-            var usage = usageStore.Model.GetCurrentUsage(ApplicationConfiguration.AssemblyName.Version.ToString(), unityVersion);
-            usage.Dimensions.Lang = CultureInfo.InstalledUICulture.IetfLanguageTag;
-            usage.Dimensions.CurrentLang = CultureInfo.CurrentCulture.IetfLanguageTag;
```

Comment:
Why remove these two lines? Is this being done somewhere else?

Reply:
Hmm... Seems I lost some code...

Reply:
Ah, cool!